### ISSUE TITLE
Resizable Dialog: NVDA reads "clickable" too many times.

### ISSUE THREAD

**Issue by v-olg** [telerik/kendo-ui-core#6472]
### Bug report

NVDA reads clickable many times in dialog.
![image](https://user-images.githubusercontent.com/77701769/124267700-490edc80-db41-11eb-9c18-f38835d39103.png)

![image](https://user-images.githubusercontent.com/77701769/124285907-12dc5780-db57-11eb-8837-23bf41b251f4.png)

### Reproduction of the problem
1. Open the [Dojo example](https://dojo.telerik.com/@NeliKondova/oqafujEZ) and turn on NVDA
 2. Click the button to open the Window
 
### Current behavior

Once the Widow gets focused and NVDA is reading, the "clickable" is repeated too many times. 
Note: Sometimes it is needed to close and open the Window again to replicate the issue.

### Expected/desired behavior
The "div" blocks that are responsible for resizing the window should be descriptive to not confuse NVDA users of many "clickable" or something else to do it more accessible.

### Environment

* **Kendo UI version:** "2021.2.616"

**Comment by v-vravu** [comment]
Same behavior is observed in Kendo.Mvc 2015.3.1111.545
